.buildkite/publish.sh

```diff
@@ -105,13 +112,37 @@ rmdir release/webasm
 pushd release
 files=()
 while IFS='' read -r line; do files+=("$line"); done < <(find . -type f | sed 's#^./##')
+backticks='```' # hack for bad Vim syntax highlighting definition
 release_notes="To use Sorbet add this line to your Gemfile:
-\`\`\`
+$backticks
 gem 'sorbet', '$release_version', :group => :development
 gem 'sorbet-runtime', '$release_version'
-\`\`\`"
+$backticks"
 if [ "$dryrun" = "" ]; then
-    echo "$release_notes" | ../.buildkite/tools/gh-release.sh sorbet/sorbet "${long_release_version}" -- "${files[@]}"
+  echo "$release_notes" | ../.buildkite/tools/gh-release.sh sorbet/sorbet "${long_release_version}" -- "${files[@]}"
 fi
 popd
 
+echo "--- releasing VS Code extension"
+pushd vscode_extension
+
+extension_release_version="$(jq --raw-output '.version' package.json)"
+echo "releasing $extension_release_version"
+
+# (progress bar messes with Buildkite timestamps)
+yarn install --no-progress
+
+if node_modules/.bin/vsce show --json sorbet.sorbet-vscode-extension | \
+    jq --raw-output '.versions[] | .version' | \
+    grep -qFx "$extension_release_version"; then
```

Comment:
`vsce show` gives us a list of all published versions, so we can early exit even in non-dry-run mode if the version in the package.json has not been changed.